project/Dependencies.scala

```diff
@@ -67,24 +67,13 @@ object Dependencies {
 
   val springCore = "org.springframework" % "spring-core" % "5.2.2.RELEASE"// Apache v2
 
-  // NB: Updated from 14.0.1 as suggested in
-  //       https://github.com/google/guava/issues/3249,
-  //       https://github.com/google/guava/issues/3345
-  //     to prevent test org.apache.toree.magic.builtin.BuiltinLoaderSpec
-  //     from failing on OpenJDK 11.
-  val guava = "com.google.guava" % "guava" % "31.0.1-jre" // Apache v2
+  // TOREE-552 Align Guava with Spark
+  val guava = "com.google.guava" % "guava" % "14.0.1" // Apache v2
 
   // Projects
-
   val sparkAll = Def.setting{
     Seq(
-      sparkCore.value % "provided" excludeAll(
-        // Exclude netty (org.jboss.netty is for 3.2.2.Final only)
```

Comment:
The recent version of Spark does not depend on Netty 3.x, so this exclusion is redundant.

https://mvnrepository.com/artifact/org.apache.spark/spark-core_2.12/3.3.2